data_steward/validation/participants/writers.py

```diff
@@ -222,8 +222,15 @@ def create_site_validation_report(project, dataset, hpo_list, bucket, filename):
 
     # reset the stream and write to the bucket
     report_file.seek(0)
-    report_result = gcs_utils.upload_object(bucket, filename, report_file)
-    report_file.close()
+    try:
+        blob.upload_from_file(report_file)
+    except GoogleCloudError as exc:
+        LOGGER.exception(
+            f"Encountered {str(exc.message)} error when writing site report")
+    except Exception as exc:
+        LOGGER.exception(f"Encountered {str(exc)}")
+        raise exc
 
-    LOGGER.info(f"Wrote validation report csv: {bucket}{filename}")
-    return report_result, read_errors
+    report_file.close()
+    LOGGER.info(f"Wrote validation report csv: {blob.bucket}/{blob.name}")
```

Comment:
Usage of `client.project`, `bucket.name` and `blob.name` when strings are required.